test/unit/node-labels.test.js

```diff
@@ -526,3 +526,21 @@ for (const info of specificTools) {
     })
   }
 }
+
+[
+  [ ['c++', 'n-api'],
+    ['src/node_api.cc', 'src/node_api.h', 'src/node_api_types.h'] ],
+  [ ['test', 'n-api'], ['test/addons-napi/foo'] ]
+].forEach((info) => {
+  const labels = info[0]
+  const files = info[1]
```

Comment:
I've been on the lazy side lately and started using `const [ labels, files ] = info`, even gives some hipster creds -- so is by no means a request for change, just thinking out loud.